napari/layers/base/base.py

```diff
@@ -1257,6 +1257,35 @@ def _display_bounding_box(self, dims_displayed: np.ndarray):
         """An axis aligned (self._ndisplay, 2) bounding box around the data"""
         return self._extent_data[:, dims_displayed].T
 
+    def click_plane_from_click_data(
```

Comment:
we should add to the 3d interactivity docs a description of things down the road, definitely not blocking merge for this!

Reply:
Agreed!

Reply:
after discussion, let's rename this to `_world_to_data_plane`